interfaces/functions.yaml

```diff
@@ -1090,6 +1159,12 @@ igraph_maximal_cliques:
     PARAMS: GRAPH graph, OUT VERTEXSET_LIST res, INTEGER min_size=0, INTEGER max_size=0
     DEPS: res ON graph
 
+igraph_maximal_cliques_subset:
+    PARAMS: |-
+        GRAPH graph, VECTOR_INT subset, OUT VECTOR_INT_LIST res,
```

Comment:
`subset` should be of type `VERTEX_INDICES` to make sure that R translates between 0-based and 1-based indices when generating code.

Similarly, `res` should be `VERTEXSET_LIST`